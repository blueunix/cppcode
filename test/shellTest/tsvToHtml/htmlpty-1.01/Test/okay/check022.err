:11:<PLAINTEXT> is abberant HTML and is strongly deprecated; use <PRE>...</PRE> instead
:13:</PLAINTEXT> is treated as ordinary text: PLAINTEXT environment continues to end-of-file
:16:non-zero <TAG> ... </TAG> nesting level at end-of-file
:16:unclosed tags in tag nest: line:opentag = 10:BODY 3:HTML
