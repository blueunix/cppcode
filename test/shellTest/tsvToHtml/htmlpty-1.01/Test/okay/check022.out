<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">

<HTML>
    <HEAD>
        <TITLE>
            <!-- Please supply a descriptive title here -->
        </TITLE>
        <LINK REV="made" HREF="mailto:beebe@plot79.math.utah.edu">
    </HEAD>
    <BODY>
<PLAINTEXT>
blah blah
</PLAINTEXT>
    </BODY>
</HTML>

